src/cephadm/cephadm

```diff
@@ -3120,6 +3144,21 @@ def command_bootstrap():
         if not mon_network:
             raise Error('Failed to infer CIDR network for mon ip %s; pass '
                         '--skip-mon-network to configure it later' % base_ip)
+    
+    cluster_network = None
+    ipv6_cluster_network = False
+    # the cluster network may not exist on this node, so all we can do is 
+    # validate that the address given is valid ipv4 or ipv6 subnet
+    if args.cluster_network:
+        rc, versions, err_msg = check_subnet(args.cluster_network)
+        if rc:
+            raise Error(f"Invalid --cluster-network parameter: {err_msg}")
+        cluster_network = args.cluster_network
+        ipv6_cluster_network = True if 6 in versions else False
+    else:
+        logger.warning("{}{}{}".format(termcolor.yellow,
+                                       "Internal network (--cluster-network) has not been provided",
+                                       termcolor.end))
```

Comment:
I wouldn't warn here--this suggests this option is recommended.  For most(?) deployments it is not.

Reply:
it's a warn so the admin understands that if they wanted to have network separation they need additional steps.